### ISSUE TITLE
Proxy Support ?

### ISSUE THREAD

**Issue by chromer030** [agourlay/dlm#6]

**Comment by chromer030** [comment]
```
user@user ~/Desktop % HTTPS_PROXY="http://127.0.0.1:10808" ./dlm --inputFile 1.txt --outputDir . --maxConcurrentDownloads 2
[1]    2655 segmentation fault (core dumped)  HTTPS_PROXY="http://127.0.0.1:10808" ./dlm --inputFile 1.txt --outputDir .  2

```

**Comment by chromer030** [comment]
Is'nt this related to musl ? what about glibc build ?

**Comment by agourlay** [comment]
The binaries are currently built and uploaded by this [Github action](https://github.com/rust-build/rust-build.action) which supports only `musl`.
My plan is to setup a different GA to get access to more targets to fix #1

**Comment by agourlay** [comment]
For your info #1 has been resolved :heavy_check_mark:
